cli/config/configuration-acrn.toml.in

```diff
@@ -224,6 +224,12 @@ disable_guest_seccomp=@DEFDISABLEGUESTSECCOMP@
 # If you are using docker, `disable_new_netns` only works with `docker run --net=none`
 # (default: false)
 #disable_new_netns = true
+#
+# if enable, the runtime use the parent cgroup of a container PodSandbox.  This
```

Comment:
> If enabled, the runtime uses the parent "PodSandbox" cgroup.

I'm afraid I got lost with the second sentence - can you break it into two sentences maybe to make it easier to understand?

Reply:
Hey @jodh-intel  I have tried to re-work the sentences, let me know if that works. I still feel is not good enough the main thing is that this option works based in container types. Specially `PodSanbox`  I wanted to point to some documentation but seems that we dont have documentation about it.